db/seeds.rb

```diff
@@ -1,249 +1,242 @@
 
-if (admin_access_token = ENV['ADMIN_ACCESS_TOKEN'])
-  access_token = user.access_tokens.build(name: 'Administration', permission: 'rw')
-  access_token.scopes = access_token.class.scopes.values
-  access_token.value = admin_access_token
-  access_token.save!
-end
 
+  ###
+  #  Creating impersonation admin user for provider
+  ###
 
-###
-#  Creating impersonation admin user for provider
-###
+  impersonation_admin = provider.users.build_with_fields
+  impersonation_admin.account = provider
+  impersonation_admin.role = :admin
+  impersonation_admin.signup_type = :minimal
+  impersonation_admin.state = 'active'
 
-impersonation_admin = provider.users.build_with_fields
-impersonation_admin.account = provider
-impersonation_admin.role = :admin
-impersonation_admin.signup_type = :minimal
+  impersonation_admin_config = ThreeScale.config.impersonation_admin
+  impersonation_admin_username = impersonation_admin_config['username']
+  impersonation_admin.attributes = {
+    username: impersonation_admin_username,
+    email: "#{impersonation_admin_username}+#{provider.self_domain}@#{impersonation_admin_config['domain']}",
+    first_name: '3scale',
+    last_name: 'Admin'
+  }
 
-impersonation_admin_config = ThreeScale.config.impersonation_admin
-impersonation_admin_username = impersonation_admin_config['username']
-impersonation_admin.attributes = {
-  username: impersonation_admin_username,
-  email: "#{impersonation_admin_username}+#{provider.self_domain}@#{impersonation_admin_config['domain']}",
-  first_name: '3scale',
-  last_name: 'Admin'
-}
+  impersonation_admin.save!
 
-impersonation_admin.save!
-impersonation_admin.activate!
+  ###
+  #  Enabling Provider Switches
+  ###
 
-###
-#  Creating Sample Data
-###
+  provider.force_upgrade_to_provider_plan!(plan)
 
-SignupWorker.enqueue(provider)
+  ###
+  # Basic enabled/disabled switches
+  ###
 
-###
-#  Enabling Provider Switches
-###
-
-provider.force_upgrade_to_provider_plan!(plan)
-
-###
-# Basic enabled/disabled switches
-###
+  Settings.basic_enabled_switches.each do |name|
+    provider.settings.public_send("show_#{name}!")
+  end
 
-Settings.basic_enabled_switches.each do |name|
-  provider.settings.public_send("show_#{name}")
-end
+  Settings.basic_disabled_switches.each do |name|
+    provider.settings.public_send("hide_#{name}!")
+  end
 
-Settings.basic_disabled_switches.each do |name|
-  provider.settings.public_send("hide_#{name}")
-end
+  # Import CMS templates
+  SimpleLayout.new(provider).import! if ENV.fetch('CMS_TEMPLATES', Rails.env.development?).presence
 
-# Import sample data in dev mode
-if sample_data and Rails.env.development?
-  provider.create_sample_data!
-  provider.update_columns(sample_data: false)
-end
+  ###
+  #  Creating Sample Data
+  ###
 
-# Import CMS templates
-SimpleLayout.new(provider).import! if ENV.fetch('CMS_TEMPLATES', Rails.env.development?).presence
+  if Rails.env.development?
+    SignupWorker::SampleDataWorker.new.perform(provider.id)
```

Comment:
Nice!

Reply:
Why not invoking `ImportSimpleLayoutWorker.new.perform(provider.id)` as well though?

Reply:
Right!! I will do it 👍 💪